Makefile

```diff
@@ -1,7 +1,8 @@
-test:
-	xcodebuild test -project "SwiftGenStrings.xcodeproj" -scheme "SwiftGenStrings" -destination "platform=macOS"
-
 release:
-	xcodebuild -scheme "SwiftGenStrings" -configuration "Release" -destination "generic/platform=macOS" clean archive -archivePath "build/"
+	swift build -c release
 	mkdir -p Products
-	cp build.xcarchive/Products/usr/local/bin/SwiftGenStrings Products/
+	cp .build/release/SwiftGenStrings Products/SwiftGenStrings
+
+install:
```

Comment:
we can depend on release and make the command a bit more intuitive:

```
install: release
    install -v Products/.. .....
```

Reply:
Very nice!